pkg/config/config.go

```diff
@@ -135,6 +135,8 @@ type PluginConfig struct {
 	SystemdCgroup bool `toml:"systemd_cgroup" json:"systemdCgroup"`
 	// EnableTLSStreaming indicates to enable the TLS streaming support.
 	EnableTLSStreaming bool `toml:"enable_tls_streaming" json:"enableTLSStreaming"`
+	// DisableHTTP2Client disables http2 client operations (fetching/pulling)
+	DisableHTTP2Client bool `toml:"disable_http2_client" json:"disableHTTP2Client"`
```

Comment:
If we want this to just affect image pulls, I think we should move this option into the `Registry` section above.

Reply:
Will do

Reply:
I'll push the deploy change on our side as well for this after